nucypher/utilities/sandbox/constants.py

```diff
@@ -135,7 +136,7 @@ def select_test_port() -> int:
 
 MOCK_REGISTRY_FILEPATH = os.path.join(BASE_TEMP_DIR, f'{BASE_TEMP_PREFIX}mock-registry-{datetime.now().strftime(DATETIME_FORMAT)}.json')
 
-TEMPORARY_DOMAIN = ':TEMPORARY_DOMAIN:'  # for use with `--dev` node runtimes
+TEMPORARY_DOMAIN = ":TEMPORARY_DOMAIN:"  # for use with `--dev` node runtimes  # FIXME?
```

Comment:
What are you thinking to fix here?

Reply:
Nothing, I just wasn't sure if this was an appropriate domain for tests, but I realized later that it's OK. Removed the FIXME.